config.json

```diff
@@ -612,6 +612,18 @@
       "unlocked_by": "rotational-cipher",
       "uuid": "d36ce010-210f-4e9a-9d6c-cb933e0a59af"
     },
+    {
+      "core": false,
+      "difficulty": 5,
+      "slug": "go-counting",
+      "topics": [
+        "algorithms",
+        "loops",
+        "conditionals"
+      ],
+      "unlocked_by": "triangle",
```

Comment:
What about adding "games" as a topic since go is a game? 🙂 

Also, scrabble-score might be a good exercise for this to be unlocked by since that is also a game and all the other game exercises are unlocked by it 🙂